docker-entrypoint.sh

```diff
@@ -1,17 +1,21 @@
 #!/bin/sh
 set -eux
 
+pageserver_id_param="${NODE_ID:-10}"
```

Comment:
That 10 looks weird, but let's keep it in case someone depends on it :)

It looks like no one used this docker image except console, because it didn't support some arguments.

Reply:
I think this is because in the console's docker-compose.yaml NODE_ID isn't set. We can add it there and remove here. Also we always run etcd now, so maybe we can remove condition on broker endpoints?

So the only condition left is presence of tenants dir, which is probably ok to put into `command` section in the docker-compose.yaml

Reply:
10 was chosen because there was a bug in regex that was handling only single digit node ids. 

This script is used in console docker-compose setup to run proxy as well as storage components so unless we can completely separate pageserver config from we need to keep these things optional. 

I think we need to refactor this part inside pageserver. See https://github.com/neondatabase/neon/issues/2192

Reply:
So for now I would rather keep the script mostly as it is. Having tenant dir condition in line is a bit messy to me